SETUP/DEVELOPMENT.md

```diff
@@ -0,0 +1,79 @@
+# Code Development
+
+This doc is designed for users who want to develop on the DProofreaders (DP)
+codebase.
+
+## Introduction
+
+The DP code is almost entirely server-side PHP with a smattering of JS and
+CSS. It uses a MySQL backend database and requires a web server -- pgdp.net
+uses Apache but Nginx should work as well. See the [installation docs](INSTALL.md)
+for information on minimum middleware versions.
+
+The code does not use a PHP framework or an ORM. It was developed 20 years ago
+and has grown very organically over the years. There has been effort in the
+past decade to standardize things but you'll find many different patterns remain.
+
+## Development Environment
+
+The best current development environment is using the
```

Comment:
Would it be useful to say something here about developing in a sandbox on the test server? I've found this to be very satisfactory. It is so responsive it is hard to believe I'm working on a machine 8000 miles away except on very rare occasions when there are server or internet problems. It seems to be quite easy to set up compared to a VM. Is there something I'm missing?

Reply:
I agree - it was relatively easy to get set up to develop using the test server, but it did require a bit of hand-holding, for me at least. A little information here would probably reduce the need for that, and maybe a direct link to https://www.pgdp.net/wiki/DP_Code_Development_Using_git which I found useful (unless I missed that in this doc).

Reply:
The TEST server is super useful but also a bit tricky. It's a shared dev environment and also houses some handy not-PROD-but-not-far-from-it resources. We don't give access to every potential dev that asks, they need to be a bit of a known entity and, by some definition "proved" themselves first.

In short: it's not a good option for someone who just stumbles over the repo.